scout/adapter/mongo/case.py

```diff
@@ -10,7 +10,7 @@
 from bson import ObjectId
 
 from scout.build.case import build_case
-from scout.constants import ACMG_MAP, FILE_TYPE_MAP, ID_PROJECTION
+from scout.constants import ACMG_MAP, FILE_TYPE_MAP, ID_PROJECTION, OMICS_FILE_TYPE_MAP
```

Comment:
It seems likely we will have more OMICSs file types in the future, be that protein expression, explicit methylation region changes or whatnot, so introducing a map constant.